apply/screening-example.js

```diff
@@ -0,0 +1,118 @@
+const averageRiskScreeningPlan = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/PlanDefinition-CervicalCancerScreeningAverageRisk.json'));
+
+// ActivityDefinition resources
+const cytologyActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-CervicalCytologyScreeningRecommendation.json'));
+const colposcopyActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-ColposcopyActivityDefinition.json'));
+const errorActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-CommunicateErrors.json'));
+const communicateActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-CommunicateUnstructuredRecommendations.json'));
+const cotestActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-CotestingScreeningRecommendation.json'));
+const dashboardActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-DisplayCervicalCancerMedicalHistory.json'));
+const hpvActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-PrimaryHpvScreeningRecommendation.json'));
+const surveillanceActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-SurveillanceActivityDefinition.json'));
+const treatmentActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-TreatmentActivityDefinition.json'));
+
+// Questionnaire resources
+const provideMoreInformationQuestionnaire = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/Questionnaire-ProvideMoreInformation.json'));
+
+// Bring in an example patient bundle from the test folder
+const examplePatientBundle = JSON.parse(readFileSync('test/ScreeningAverageRisk/test_results/ScreeningAverageRiskLibrary_v1.0.0/bundles/Under_30_and_have_had_recent_cytology_test.json'));
+const examplePatientResources = examplePatientBundle.entry.map(r => r.resource); // convert from bundle to flat array
+
+// Read in ELM JSON representation of CDS logic
+const elmJsonDependencyArray = [
+  JSON.parse(readFileSync('cql/TopLevelScreeningLibrary.json')),
+  JSON.parse(readFileSync('cql/ScreeningSymptomaticLibrary.json')),
+  JSON.parse(readFileSync('cql/ScreeningDesExposureLibrary.json')),
+  JSON.parse(readFileSync('cql/ScreeningImmunocompromisedLibrary.json')),
+  JSON.parse(readFileSync('cql/ScreeningAverageRiskLibrary.json')),
+  JSON.parse(readFileSync('cql/DisplayCervicalCancerMedicalHistory.json')),
+  JSON.parse(readFileSync('cql/CCSMCommonFunctions.json')),
+  JSON.parse(readFileSync('cql/CDSConnectCommonsforFHIRv401.json')),
+];
+
+// Reformat ELM JSON value set references to match what is expected by the 
+// code service built into the cql execution engine
+const elmJsonDependencies = elmJsonDependencyArray.reduce((acc, elm) => {
```

Comment:
Should the value set reference reformatting be its own function somewhere? This seems like a step we would have to repeat for every test?